src/Telegram.Bot/TelegramBotClient.cs

```diff
@@ -744,14 +852,16 @@ public Task<User> GetMeAsync(CancellationToken cancellationToken = default)
             int replyToMessageId = default,
             IReplyMarkup replyMarkup = default,
             CancellationToken cancellationToken = default,
-            string vCard = default
+            string vCard = default,
+            bool allowSendingWithoutReply = default
```

Comment:
The default convention in .NET world is to setup `CancellationToken` at the end of the calling params.